rasa/telemetry.py

```diff
@@ -628,50 +645,249 @@ async def track_model_training(
+        },
     )
 
 
 @ensure_telemetry_enabled
-async def track_telemetry_disabled() -> None:
+def track_telemetry_disabled() -> None:
     """Track when a user disables telemetry."""
+    _track(TELEMETRY_DISABLED_EVENT)
+
+
+@ensure_telemetry_enabled
+def track_data_split(fraction: float, data_type: Text) -> None:
+    """Track when a user splits data.
+
+    Args:
+        fraction: How much data goes into train and how much goes into test
+        data_type: Is this core, nlu or nlg data
+    """
+    _track(TELEMETRY_DATA_SPLIT_EVENT, {"fraction": fraction, "type": data_type})
+
+
+@ensure_telemetry_enabled
+def track_validate_files(validation_success: bool) -> None:
+    """Track when a user validates data files.
+
+    Args:
+        validation_success: Whether the validation was successful
+    """
+    _track(TELEMETRY_DATA_VALIDATED_EVENT, {"validation_success": validation_success})
+
+
+@ensure_telemetry_enabled
+def track_data_convert(output_format: Text, data_type: Text) -> None:
+    """Track when a user converts data.
+
+    Args:
+        output_format: Target format for the converter
+        data_type: Is this core, nlu or nlg data
+    """
+    _track(
+        TELEMETRY_DATA_CONVERTED_EVENT,
+        {"output_format": output_format, "type": data_type},
+    )
+
+
+@ensure_telemetry_enabled
+def track_tracker_export(
+    number_of_exported_events: int,
+    tracker_store: "TrackerStore",
+    event_broker: Optional["EventBroker"],
+) -> None:
+    """Track when a user exports trackers.
 
-    asyncio.ensure_future(track(TELEMETRY_DISABLED))
+    Args:
+        number_of_exported_events: Number of events that got exported
+        tracker_store: Store used to retrieve the events from
+        event_broker: Broker the events are getting published towards
+    """
+    _track(
+        TELEMETRY_TRACKER_EXPORTED_EVENT,
+        {
+            "number_of_exported_events": number_of_exported_events,
+            "tracker_store": type(tracker_store).__name__,
+            "event_broker": type(event_broker).__name__ if event_broker else None,
+        },
+    )
+
+
+@ensure_telemetry_enabled
+def track_interactive_learning_start(
+    skip_visualization: bool, save_in_e2e: bool
+) -> None:
+    """Track when a user starts an interactive learning session.
+
+    Args:
+        skip_visualization: Is visualization skipped in this session
+        save_in_e2e: Is e2e used in this session
+    """
+    _track(
+        TELEMETRY_INTERACTIVE_LEARNING_STARTED_EVENT,
+        {"skip_visualization": skip_visualization, "save_in_e2e": save_in_e2e},
+    )
+
+
+@ensure_telemetry_enabled
+def track_server_start(
+    input_channels: List["InputChannel"],
+    endpoints: Optional["AvailableEndpoints"],
+    model_directory: Optional[Text],
+    number_of_workers: int,
+    is_api_enabled: bool,
+) -> None:
+    """Track when a user starts a rasa server.
+
+    Args:
+        number_of_workers: number of used Sanic workers
```

Comment:
The order of the function args and the docstring description should be the same imo